feat: added buttons for Tailwind code options

Abbeyafolaby opened the pull request:
"re #409"

# Fixes Issue

**My PR closes #409 **

# 👨‍💻 Changes proposed(What did you do ?)
Added Tailwind buttons to the Result Pages and  functions that show a popup when clicked


# ✔️ Check List (Check all the applicable boxes)
<!-- Mark all the applicable boxes. To mark the box as done follow the following conventions -->

<!--
[x] - Correct; marked as done

[ ] - Not correct; marked as **not** done
-->

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.

##  Note to reviewers

<!-- Add notes to reviewers if applicable -->

# 📷 Screenshots

<!-- Add all the screenshots which support your changes -->
![Screenshot (84)](https://github.com/Dun-sin/Code-Magic/assets/91214759/0db8837d-1a5d-4cdc-b1cd-a8db5bcd4008)

Dun-sin commented:
Looks good on visuals but why not save space and make the two buttons on the same line

Dun-sin commented on src/pages/gradient-text.ts:
```diff
@@ -214,3 +216,16 @@ function getValues() {
 resetValues();
 getValues();
+
+// Tailwind codecopy handler
+function tailwindHandler() {
+  getTailwindCode.addEventListener('click', () => {
+    showPopup(
+      'Tailwind Code Copied',
+      'Code has been successfully copied to clipboard',
+      'success'
+    );
+  });
+}
+
+tailwindHandler();
```
not reusable code, we already have one here, that's what i meant by track where the CSS code copy was for, which meant create a similar function called copyTailwindCodeToClipboard, then rename this one to copyCSSCodeToClipboard. call the act on css generator(which you will rename too and create a new function for the tailwind with the switch statement without the code under it.
https://github.com/Dun-sin/Code-Magic/blob/6ec52511508cde36b3c9cf96a5cc991aef0d1d5b/src/lib/packages.ts#L67

Here's the act on generator function:
https://github.com/Dun-sin/Code-Magic/blob/6ec52511508cde36b3c9cf96a5cc991aef0d1d5b/src/lib/packages.ts#L95-L251

Dun-sin replied:
in other words duplicate the actongenerator function change the name and make the codeToCopy instances to be an empty string for all cases.

Dun-sin commented on src/pages/gradient-text.ts:
```diff
@@ -40,6 +41,7 @@ const getRemoveColorButtonElement = getRemoveNewColorButton(attribute);
 const resetButton = getResetButton(attribute);
+const getTailwindCode = getTailwindButton(attribute);
 
 function copyHandler() {
```
we also already have a copy handler, once again you can just duplicate for the tailwind, makes this easier to understand

Abbeyafolaby commented:
Hi @Dun-sin , I have made the changes you requested for.
The elements were throwing unused var errors, so I disabled it,. should I just delete them or just leave them when you set the values so you can remove them yourselves?
![Screenshot (85)](https://github.com/Dun-sin/Code-Magic/assets/91214759/61efabe7-05e5-4711-aca3-5b2e45a446de)

Dun-sin commented:
> Hi @Dun-sin , I have made the changes you requested for. The elements were throwing unused var errors, so I disabled it,. should I just delete them or just leave them when you set the values so you can remove them yourselves? ![Screenshot (85)](https://user-images.githubusercontent.com/91214759/254709989-61efabe7-05e5-4711-aca3-5b2e45a446de.png)

just delete them and leave the empty cases

Abbeyafolaby commented:
![Screenshot (86)](https://github.com/Dun-sin/Code-Magic/assets/91214759/9095c4ac-1993-4b3e-a50b-8867259874c4)
I fixed the issue for visuals and I noticed it wasn't responsive so I added a style to it. hope that is okay?

Dun-sin reviewed (approved):
Looks good now, thanks, consider giving this project a star if you haven't

Dun-sin commented:
:tada: This PR is included in version 2.57.0 :tada:

The release is available on [GitHub release](https://github.com/Dun-sin/Code-Magic/releases/tag/v2.57.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
